### DISCUSSION TITLE
Multi-User / Multi-Session support

### DISCUSSION THREAD

**Discussion by che85** [Project-MONAI/MONAILabel#511] [General]
The readme is stating "initial support for multiple users is restricted."

For 3D Slicer on a client and monailabel implemented on a server, will there be possible collisions happening when multiple clients try running inference?

Thank you,
Christian

**Comment by SachidanandAlle** [comment]
Not anymore.. unless you have gpu available.. the readme needs update.. coming version..  we vcan highlight it better

**Reply by hw-ju** [reply]
@SachidanandAlle  Can multiple Slicer clients use a single server simultaneously now? Will there be conflicts, e.g., annotating on the same image simultaneously? Thanks!

**Comment by SachidanandAlle** [comment]
Yes.. multiple clients can work simultaneously.. the current random strategy has additional weights based on last access timestamp... so if you have enough unlabelled images to annotate.. each client will get different image for annotation.. until the full rotation happens based on weighted random probability..
